CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## v1.13.0
+
+- [core] add API to filter contributions at runtime [#9317](https://github.com/eclipse-theia/theia/pull/9317) Contributed on behalf of STMicroelectronics
```

Comment:
Is there a strong requirement to include `contributed on behalf...` in the changelog, it does not follow the format of existing entries. In the past it was simply referenced in the pull-request description itself, and/or in the commit.

Reply:
Yes it is

Reply:
This was previously agreed-upon, as one of the ways to compensate, for the lost of visibility for sponsoring contribution, compared to the "old" way of adding a mention of said contributions, as part of the copyright header of each file. 

Another permissible way is to add a similar mention in the body of commit messages.